methcomp/blandaltman.py

```diff
@@ -11,33 +12,18 @@ class _BlandAltman(object):
     """Internal class for drawing a Bland-Altman plot"""
```

Comment:
Note: in python3 `class _BlandAltman:` is the same as `class _BlandAltman(object):`. The former is cleaner...

Reply:
removed (object)